src/commands/env/env-get.js

```diff
@@ -27,7 +42,11 @@ const envGet = async (name, options, command) => {
   }
 
   if (!value) {
-    log(`Environment variable ${name} not set for site ${siteData.name}`)
+    log(
+      `No value set in the ${chalk.magentaBright(context)} context for environment variable ${chalk.yellowBright(
```

Comment:
I haven't tested yet but do we need a separate message or need to add to this message for when the scope flag is used?

Reply:
Great suggestion! I'll add this